packages/run-python-kit/package.json

```diff
@@ -0,0 +1,111 @@
+{
+  "name": "@google-labs/run-python-kit",
```

Comment:
What about just calling it Python Kit? Also let's use the new namespace, so maybe `@breadboard-ai/python-kit`?

Reply:
I have another idea. This particular kit is a no-op, since its implementation is entirely in on the node proxy server. So maybe we just reuse your runPython node and not define it as a separate kit?

Reply:
Good point. I don't think this even needs to be landed at all.